defaults/azure_config.yaml

```diff
@@ -0,0 +1,40 @@
+# TODO: Go over it and fix detauls
+instance_provision: 'spot'
+spot_max_price: 0.60
+instance_provision_fallback_on_demand: false
+region_name:
+  - westeurope
+user_credentials_path: '~/.ssh/scylla-qa-ec2'
+
+azure_instance_type_db: 'Standard_L8s_v2'
+azure_instance_type_loader: 'Standard_F4s_v2'
+azure_instance_type_monitor: 'Standard_D2_v5'
+# TODO: Make loader and monitor images
+azure_image_loader: 'OpenLogic:CentOS:7.5:latest'
+azure_image_monitor: 'OpenLogic:CentOS:7.5:latest'
```

Comment:
Latest CentOS 7 by Rogue Wave Software (formerly OpenLogic) is 7.9

Is it intended to stick with 7.5?

Reply:
it's not intended. Updating it to 7.9